### PR TITLE
fix: Tegra, Jetpack 5: add Jetson as working target

### PR DESCRIPTION
**Pull request by TheYoctoJester**
This PR is a rebased replica of https://github.com/OE4T/meta-mender-community/pull/19.

It carries the last state as of https://github.com/OE4T/meta-mender-community/pull/19#issuecomment-2254160545 comment, with the following additions:
- https://github.com/OE4T/meta-mender-community/pull/19#discussion_r1686468568 has been addressed
- https://github.com/OE4T/meta-mender-community/pull/19#discussion_r1686514205 has been addressed
- https://github.com/OE4T/meta-mender-community/pull/19#discussion_r1686614039 has been addressed
- the PR branch is rebased onto https://github.com/OE4T/meta-mendercommunity/commit/e034e61737ee5cf781a450bb4491b52bacaf7eca, which mirrors the current state of https://github.com/mendersoftware/meta-mender-community (branch kirkstone) too, so once it is merged it can be pushed upstream to that also
- the various commits by @mwest90 where author and sign-off weren't matching were author-rewritten to the signed-off-by.

### CODE REVIEW

**Comment by TheYoctoJester** [comment]
@maxekman I have removed the commit fa468f5cfe11be733564e2f2071e53cbe60a8534 which is missing the Signed-off-by: in this branch. Please let me know if you'd like to re-submit it or if I should just replicate the override removal and set you as Reported-by:?

**Review by mwest90 — COMMENTED** [review]
@TheYoctoJester looks good to me!

**Comment by maxekman** [comment]
Thanks @TheYoctoJester, yes please set me as reported by on https://github.com/mendersoftware/meta-mender-community/commit/fa468f5cfe11be733564e2f2071e53cbe60a8534. I’m on vacation right now.

**Comment by TheYoctoJester** [comment]
> Thanks @TheYoctoJester, yes please set me as reported by on [fa468f5](https://github.com/mendersoftware/meta-mender-community/commit/fa468f5cfe11be733564e2f2071e53cbe60a8534). I’m on vacation right now.

Works for me, thanks!

**Review by alfrunes — APPROVED** [review]

File: meta-mender-tegra/meta-mender-tegra-jetpack5/recipes-bsp/uefi/edk2-firmware-tegra_%.bbappend [hunk 1]
```diff
@@ -1,9 +1 @@
-
-FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
-
-SRC_URI += "file://0007-runtime-access-for-KernelCommandLine-efivar.patch;patchdir=.."
```
Comment (akfaro) [inline meta-mender-tegra/meta-mender-tegra-jetpack5/recipes-bsp/uefi/edk2-firmware-tegra_%.bbappend:4]: Sorry for being late to the party. I hope, you don't mind me posting my question here. 

I figured that mender uses the KernelCommandLine variable for passing the machine ID to the "new" RootFS if the feature "mender-persist-systemd-machine-id" is active.
Is this feature supposed to work despite removing this patch?

Upstream, the variable is still not available to runtime:
https://github.com/NVIDIA/edk2-nvidia/blob/r36.4.0/Silicon/NVIDIA/Drivers/NvidiaConfigDxe/NvidiaConfigHii.vfr#L63

Without  runtime access to this variable, the script `efi_systemd_machine_id.sh` seems to fail persisting the machine ID with the message
```
efivar: Read-only file system
```

(I tested the script on a Jetpack 6.1 system, but I don't see a relevant difference to JP 5 code)

**Pull request merged**